specification/appendix/metadata_example.md

```diff
@@ -0,0 +1,235 @@
+# Metadata Examples
+
+The following is an example metadata JSON structure provided by a hypothetical FOCUS data provider. This example also illustrates an example of how a provider can supply the required reference between the FOCUS data and the schema metadata.  Provider implementations will vary on how the metadata is stored and retrieved, however the providers chosen metadata delivery approach should be able to support the structure represented in this example. 
+
+## Scenario
+In this example, the provider supports delivery of FOCUS data via file export to a data storage system. The provider delivers data every 12 hours. 
+
+#### Example Data Structure
+
+- export root location: `/FOCUS`
```

Comment:
Maybe provide a version in the paths?

Reply:
That would be up to the providers implementation.